k8s/helm-chart/dynatrace-gcp-function/Chart.yaml

```diff
@@ -16,4 +16,13 @@ apiVersion: v2
 name: dynatrace-gcp-function
 description: A Helm chart for dynatrace-gcp-function which provides the mechanism to pull all Google Cloud metrics and/or Google Cloud logs into Dynatrace.
 type: application
-version: 0.2.0
\ No newline at end of file
+home: https://github.com/dynatrace-oss/dynatrace-gcp-function
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 1.4.0
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "1.17.0"
```

Comment:
You are right. Generally, the chartVersion should be updated each time you change the chart. Not sure how Travis should handle this. On the other hand, the app version should be filled by Travis. Maybe the solution would be to just have the same version for both handled by Travis. I'll create an improvement for that since it does exceed the scope of this story. The file will be reverted.